recipes/lyra/all/test_package/CMakeLists.txt

```diff
@@ -1,17 +1,11 @@
-# Copyright Rene Rivera 2019
```

Comment:
I don't think we can remove author's copyright.

Reply:
from CMakeLists?

Reply:
I am not a lawyer, hard to say for me. /cc @grafikrobot for his opinion.

Reply:
I can restore this copyright. It's just weird that it was here, because this CMakeLists (which is like any other CMakeLists in CCI) was written in the context of CCI, which is MIT. For test_package copied as is or deeply inspired from existing code, I can understand.

Reply:
Having multiple licenses has it's own draw backs for the owner (ie Conan). The repository is not clearly labeled as requiring BSL-1, they could be held at fault for not respecting the license.

 It also make my job painful, my employer would need to

> The copyright notices [...] must be included in all copies of the Software [... and] all derivative works of the Software, unless such copies or derivative works are solely in the form of machine-executable object code generated

Which having not of known this one file was present would be a problem since it would not be listed.

I strongly encourage you to removed this copyright, since you did agree to have it licensed under MIT, there's no need for the overlaying of permissions/requirements.

Reply:
> And since it's my copyright, it can only be me that removes it.
That's true. copyright restored

Reply:
> If you comply with MIT then you already comply with BSL

Sadly there is a different copyright for Conan than yourself.

Regardless we are all on the same book/chapter. I appreciate you removing your copyright :clap: Thank you!